src/main/scala/org/ergoplatform/network/ErgoNodeViewSynchronizer.scala

```diff
@@ -656,6 +656,9 @@ class ErgoNodeViewSynchronizer(networkControllerRef: ActorRef,
         deliveryTracker.setInvalid(pmod.id, pmod.modifierTypeId)
         penalizeMisbehavingPeer(remote)
         false
+      case Failure(e) if e.isInstanceOf[RecoverableModifierError] =>
+        context.system.eventStream.publish(RecoverableFailedModification(pmod, e))
+        false
```

Comment:
We should recover ASAP here, there is no reason to pass this modifier further to be applied. Neither of the recoverable errors should imho lead to modifier application, it should be simply skipped and set as `Unknown` ASAP.